### ISSUE TITLE
example.py forward pass output

### ISSUE THREAD

**Issue by AdgDean** [SHARANR26/Med-Palm2#3]
Hi everyone, I tried to run the example.py, but I'm getting this error during the forward pass. It seems like there's something misconfigured or a compatibility issue. If anyone has been through this process, I would like to know what the solution is. Here, I'm sharing the output when running example.py

Thank you very much, regards.

``text_config_dict` is provided which will be used to initialize `CLIPTextConfig`. The value `text_config["id2label"]` will be overriden.
`text_config_dict` is provided which will be used to initialize `CLIPTextConfig`. The value `text_config["bos_token_id"]` will be overriden.
`text_config_dict` is provided which will be used to initialize `CLIPTextConfig`. The value `text_config["eos_token_id"]` will be overriden.
Tokenized image: torch.Size([1, 3, 224, 224])
Error during forward pass: Sizes of tensors must match except in dimension 2. Expected size 8 but got size 16384 for tensor number 1 in the list.
Text: This is a short sequence.
Number of tokens: 8
Output size: None`

**Comment by hajkeoadf** [comment]
Same. Have you figured out?

**Comment by AdgDean** [comment]
No, But at this time in 2025, med-LLama o even GPT or Claude can do a great job on this matter.

**Comment by hajkeoadf** [comment]
Yeah. Thank you! I will try other models.
